pkg/products/monitoringcommon/alertmanagerConfig.go

```diff
@@ -190,7 +192,7 @@ func ReconcileAlertManagerSecrets(ctx context.Context, serverClient k8sclient.Cl
 	configSecret := &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      config.AlertManagerConfigSecretName,
```

Comment:
The name of the Secret needs to be changed to `alertmanager-<alertmanager-name>` (which in our case is `alertmanager-rhoam`) because we are unable to override the Alertmanager's .spec.configSecret with OBO. See the API [docs](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.AlertmanagerSpec) for more info about the default configSecret name.

Additionally, we should probably move all the alertmanager [config values](https://github.com/integr8ly/integreatly-operator/blob/rhoam-v1.36.0/pkg/config/observability.go#L14-L18) out of `pkg/config/observability.go` and put them in `pkg/config/monitoring.go` instead.

Reply:
Once #3249 is merged, the constant `config.AlertManagerConfigSecretName` can be moved to [pkg/config/obo.go](https://github.com/integr8ly/integreatly-operator/pull/3249/files#diff-984212e2b097f414b48d126918cc96f447df4496c7d70f1219e54d53830d8f43R3-R5) and changed to `"alertmanager-rhoam"`. Also the namespace will need to be changed to `redhat-rhoam-operator-observability`, this [helper function](https://github.com/integr8ly/integreatly-operator/pull/3249/files#diff-984212e2b097f414b48d126918cc96f447df4496c7d70f1219e54d53830d8f43R7-R9) can also be used once #3249 is merged.
```suggestion
			Name:      config.AlertManagerConfigSecretName,
			Namespace: config.GetOboNamespace(installation.Namespace),
```

CC @MStokluska

Reply:
Thanks Carl, so following this, the name will be "alertmanager-rhoam-alertmanager" ?

Reply:
I've pushed an update, let me know what you think.